docs/source/install-script.rst

```diff
@@ -0,0 +1,141 @@
+:orphan: true
+
+.. role:: bash(code)
+   :language: bash
+
+=================
+Firedrake Install
+=================
+
+The structure of the Firedrake install process consists of the following steps:
+
+1. Installing system dependencies
+2. Building and configuring PETSc
+3. Installing Python dependencies  
+
+These steps are only supported for for Linux and MacOS distributions. 
+Further we will take the above steps and discuss them in detail. 
+
+Installing system dependencies
+-------------------------------
+
+Linux
+~~~~~
+
+For Linux distributions the system dependencies are installed via the :bash:`apt-get` package manager. The necessary dependencies for running default firedrake are:
```

Comment:
You should probably say here that if your distribution does not use `apt-get` then you can pass the `--no-package-manager` flag but then it is your responsibility to ensure that these dependencies are installed.

Reply:
Ok I will make the modification. 
Regarding the system dependencies, I also saw this issue. Do you have any ideas on how to make this information not redundant?